modules/programs/autorandr.nix

```diff
@@ -181,10 +229,15 @@ let
     output ${name}
     ${optionalString (config.position != "") "pos ${config.position}"}
     ${optionalString config.primary "primary"}
+    ${optionalString (config.dpi != null) "dpi ${toString config.dpi}"}
     ${optionalString (config.gamma != "") "gamma ${config.gamma}"}
     ${optionalString (config.mode != "") "mode ${config.mode}"}
     ${optionalString (config.rate != "") "rate ${config.rate}"}
     ${optionalString (config.rotate != null) "rotate ${config.rotate}"}
+    ${optionalString (config.transform == null && config.scale != null) (
+      (if config.scale.method == "factor" then "scale" else "scale-from") +
+      " ${toString config.scale.x}x${toString config.scale.y}"
```

Comment:
Please put `+` at the beginning of the line instead of end of previous line. To match the style elsewhere.